tests/test_opds_import.py

```diff
@@ -519,10 +519,10 @@ def test_resources_are_mirrored_on_import(self):
         # Each resource was 'mirrored' to an Amazon S3 bucket.
         # The first resource has no bibframe provider in OPDS so it uses the importer's data source.
         eq_(
-            ['http://s3.amazonaws.com/test.content.bucket/Library%20Simplified%20Open%20Access%20Content%20Server/Gutenberg%20ID/10557.epub',
-             'http://s3.amazonaws.com/test.cover.bucket/Gutenberg/Gutenberg%20ID/10441/cover_10441_9.png.svg', 
+            ['http://s3.amazonaws.com/test.content.bucket/Library%20Simplified%20Open%20Access%20Content%20Server/Gutenberg%20ID/10557/Johnny%20Crow%27s%20Party.epub.images',
+             'http://s3.amazonaws.com/test.cover.bucket/Gutenberg/Gutenberg%20ID/10441/cover_10441_9.png', 
```

Comment:
Agreed that this isn't great. Could you make it work for .epub3 and leave the .images/.noimages thing alone to be handled specially by the Gutenberg-specific code?